xrpl/models/requests/ledger_entry.py

```diff
@@ -66,6 +66,29 @@ class Credential(BaseModel):
     """The type of the credential, as issued."""
 
 
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class Delegate(BaseModel):
+    """
+    Required fields for requesting a Delegate ledger object if not querying by
+    object ID.
+    """
+
+    account: str = REQUIRED  # type: ignore
+    """
+    This field is required.
+
+    :meta hide-value:
+    """
+
+    authorize: str = REQUIRED  # type: ignore
+    """
+    This field is required.
```

Comment:
Could you add description for this field?

Reply:
added description in 4b1491bc